content/blog/usethis-2-0-0/index.Rmd

```diff
@@ -0,0 +1,258 @@
+It is a better security practice anyway to avoid storing such secrets in plain text, whenever possible.
+
+```
+BLAH=foo
+GITHUB_PAT=xyz  # <-- WE SUGGEST YOU REMOVE THIS LINE FROM .Renviron
+WUT=yo
+OTHER_WEB_SERVICE=super-secret-very-powerful-token
+```
+
+The `host`-specific PAT is now retrieved from the Git credential store upon first need.
+Note that the PAT **is** still cached in an environment variable for reuse during the remainder of the current R session.
+
+Our full recommendations for getting and configuring a PAT are in the new vignette [Managing Git(Hub) Credentials](https://usethis.r-lib.org/articles/articles/git-credentials.html).
+
+### Git protocol
+
+When usethis configures a new Git remote, it must choose a protocol, either HTTPS or SSH.
+The new default in this situation is HTTPS, because we think HTTPS is a better choice, for most users and projects (rationale is above).
+
+```{r}
+git_protocol()
+```
+
+In relevant functions, such as `use_github()`, you can still specify the `protocol` directly.
+And, as always, you can specify the protocol for a session with `use_git_protocol()` or for all sessions via the `usethis.protocol` option.
+Those who prefer SSH may want to set this option in `.Rprofile` going forward.
+
+## Pull request helpers
+
+The team that maintains the tidyverse and r-lib packages makes heavy use of GitHub pull requests for managing internal and external contributions.
+The `pr_*()` family of functions supports pull request workflows, for maintainers and contributors.
+This family has gained a couple of functions and some existing functions are more helpful.
+
+New pull request functions:
+
+* `pr_resume()` resumes work on an existing local PR branch. It can be called
+  argument-less, to select a branch interactively.
+  
+    ```{r, eval = FALSE}
+    > pr_resume()
+    ℹ No branch specified ... looking up local branches and associated PRs
+    Which branch do you want to checkout? (0 to exit) 
+    
+    1: avalcarcel9-add_use_author --> #833 ('@avalcarcel9'): Add use author
+    2:                 latex-hell
+    3:                     holder
+    4:            patch-for-withr
+    
+    Selection: 
+    ```
+  
+* `pr_forget()` abandons a PR you initiated locally or fetched from GitHub. It
+  only does local clean up and will never delete a remote branch or close a PR.
+  
+`pr_fetch()` and `pr_view()` also present an interactive choice, when the target PR is not specified or implied.
+
+## Other goodies
+
+Here's a selection of other new features:
+
+  * A default Git branch named `main` now works. `git_branch_default()` is a new
+    function that tries to discover the default branch from the local or remote
+    Git repo. Internally, it is used everywhere that we previously assumed a
+    default branch named `master`.
+  * `use_github_pages()` and `use_tidy_pkgdown()` are great for turning on
```

Comment:
Probably shouldn't promote `use_tidy_pkgdown()` because it's our workflow?

Reply:
In this case, `use_tidy_pkgdown()` is both our workflow AND what I would recommend for other maintainers as well. `use_tidy_pkgdown()` is `use_pkgdown() + use_github_pages() + use_github_action("pkgdown") + use_pkgdown_url()`.

Not sure what to do about that, if anything. I see your point, i.e. where other "tidy" functions often do something that is overkill or inappropriate for others. But it's not the case here.

Reply:
In any case, it's not really a blog post problem but potentially a function naming problem.

Reply:
Yeah, feels like we might've given the wrong name. Maybe should be `use_github_pkgdown()` or similar?

Reply:
https://github.com/r-lib/usethis/issues/1308